composeApp/src/commonMain/kotlin/band/effective/office/elevator/components/TimePickerModal.kt

```diff
@@ -27,13 +27,14 @@ import kotlinx.datetime.LocalTime
 
 @Composable
 fun TimePickerModal(
+    startTime: LocalTime = getCurrentTime(),
     titleText: String,
     modifier: Modifier = Modifier,
     onClickCansel: () -> Unit,
     onClickOk: (LocalTime) -> Unit
 ) {
     val selectorSettings = WheelPickerDefaults.selectorProperties(enabled = false)
-    var currentTime = getCurrentTime()
+    var time = startTime
```

Comment:
is this a fix for a timing bug? There the bug manifests itself when the picker scrolls quickly. `snap effect` doesn't have time to work there. In the callback is triggered to change the time

Reply:
I don't know about it bug. Fixed display of the selected time rather always current